src/Ocelot/Configuration/DownstreamRoute.cs

```diff
@@ -41,7 +41,8 @@ public class DownstreamRoute
             string downstreamHttpMethod,
             Version downstreamHttpVersion,
             HttpVersionPolicy downstreamHttpVersionPolicy,
-            Dictionary<string, UpstreamHeaderTemplate> upstreamHeaders)
+            Dictionary<string, UpstreamHeaderTemplate> upstreamHeaders,
+            MetadataOptions metadataOptions)

```

Comment:
If we extend the constructor, why no `readonly` for the class definition?

Reply:
Well, still following the current logic, I don't get that too.

Reply:
I was mad, for sure.